styles/login.css

```diff
@@ -0,0 +1,35 @@
+.form-login{
+    z-index: 3;
+}
+
+.form-login input {
+    padding: 0.5rem;
+    font-size: 1rem;
+    border: 0.2rem solid var(--color-primary);
+    background: var(--color-white);
+    border-radius: 5px;
+}
+
+.main-login{
+    display:flex;
+    justify-content: flex-end;
+}
+
+.card-vertical{
+    gap:0.4rem;
+    height:30rem;
+    padding:1rem;
+    margin: 2rem 2rem 30rem 0; 
+}
+
+.main-login .terms{
+    color:dodgerblue;
```

Comment:
you can use a variable here

Reply:
yes Jalaj. Pushed the change.
Thank you very much for the review.